lib/oci_utils/__init__.py

```diff
@@ -96,5 +96,13 @@ def _set_proxy():
         pass
 
 
+def _setup_env():
+    """
+    protect ourself for sub process executions
+    """
+    os.environ['LC_ALL'] = 'C'
```

Comment:
None of command executions are protected against langage change nor missing locale. 
As such they may be failing to parse outputs or to execute . This is what happen in the current KVM image as locals are missing.
It is a quick fix for this. Otherwise we have to go though all command execution in the code.